tests/test_auto/test_tune.py

```diff
@@ -0,0 +1,210 @@
+from os import unlink
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+import pytest
+from optuna.distributions import CategoricalDistribution
+from optuna.distributions import DiscreteUniformDistribution
+from optuna.distributions import IntLogUniformDistribution
+from optuna.distributions import IntUniformDistribution
+from optuna.distributions import LogUniformDistribution
+from optuna.distributions import UniformDistribution
+from optuna.storages import RDBStorage
+from typing_extensions import Literal
+from typing_extensions import NamedTuple
+
+from etna.auto import Tune
+from etna.auto.auto import AutoBase
+from etna.auto.auto import _Callback
+from etna.auto.auto import _Initializer
+from etna.metrics import MAE
+from etna.models import NaiveModel
+from etna.models import SimpleExpSmoothingModel
+from etna.pipeline import Pipeline
+from etna.transforms import AddConstTransform
+
+
+@pytest.fixture()
+def optuna_storage():
```

Comment:
If we use the same fixtures here as in `test_auto`, let's extract them into `conftest.py`.